### ISSUE TITLE
thread 'main' panicked at 'unsupported custom section: 'linking''

### ISSUE THREAD

**Issue by xor** [koute/cargo-web#81]
Trying to run the minimal example I get:
```
[xor@desktop wasm]$ RUST_BACKTRACE=1 cargo-web start --target wasm32-unknown-unknown
warning: debug builds on the wasm32-unknown-unknown are currently totally broken
         forcing a release build
   Compiling dtoa v0.4.2
   Compiling itoa v0.3.4
   Compiling futures v0.1.18
   Compiling serde v1.0.29
   Compiling num-traits v0.2.1
   Compiling base-x v0.2.2
   Compiling stdweb v0.4.1
   Compiling unicode-xid v0.1.0
   Compiling proc-macro2 v0.2.3
   Compiling quote v0.4.2
   Compiling syn v0.12.13
   Compiling serde_json v1.0.10
   Compiling serde_derive_internals v0.20.0
   Compiling serde_derive v1.0.29
   Compiling stdweb-internal-macros v0.1.0
   Compiling stdweb-derive v0.4.0
   Compiling wasm v0.1.0 (file:///home/xor/workspace/rust/wasm)
    Finished release [optimized] target(s) in 55.60 secs
    Garbage collecting "wasm.wasm"...
    Processing "wasm.wasm"...
thread 'main' panicked at 'unsupported custom section: 'linking'', src/wasm_context.rs:631:25
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at libstd/sys_common/backtrace.rs:59
             at libstd/panicking.rs:206
   3: std::panicking::default_hook
             at libstd/panicking.rs:222
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:400
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:347
   6: cargo_web::wasm_context::Context::from_module
   7: cargo_web::wasm::process_wasm_file
   8: core::ops::function::impls::<impl core::ops::function::FnMut<A> for &'a mut F>::call_mut
   9: cargo_web::cargo_shim::BuildConfig::build_internal
  10: cargo_web::cargo_shim::BuildConfig::build
  11: cargo_web::build::Project::build
  12: cargo_web::cmd_start::LastBuild::new
  13: cargo_web::cmd_start::command_start
  14: cargo_web::main
  15: std::rt::lang_start::{{closure}}
  16: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:305
  17: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
  18: std::rt::lang_start_internal
             at libstd/panicking.rs:284
             at libstd/panic.rs:361
             at libstd/rt.rs:58
  19: main
  20: __libc_start_main
  21: _start
```
```
[xor@desktop wasm]$ cargo web -V
cargo-web 0.6.8
```
```
[xor@desktop wasm]$ rustup show
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu (default)

installed targets for active toolchain
--------------------------------------

wasm32-unknown-emscripten
wasm32-unknown-unknown
x86_64-unknown-linux-gnu

active toolchain
----------------

nightly-x86_64-unknown-linux-gnu (default)
rustc 1.26.0-nightly (2789b067d 2018-03-06)
```

**Comment by jsheard** [comment]
You need to update, that problem was fixed in 0.6.9

`cargo install --force cargo-web`

**Comment by xor** [comment]
With the updated version of cargo-web it works now, must have been it indeed.
